sdk/spring/spring-messaging-azure/src/main/java/com/azure/spring/messaging/implementation/config/AzureListenerEndpointRegistry.java

```diff
@@ -141,11 +141,10 @@ protected MessageListenerContainer createListenerContainer(AzureListenerEndpoint
         }
 
         int containerPhase = listenerContainer.getPhase();
-        if (containerPhase < Integer.MAX_VALUE) {  // a custom phase value
-            if (this.getPhase() < Integer.MAX_VALUE && this.getPhase() != containerPhase) {
-                throw new IllegalStateException("Encountered phase mismatch between container factory definitions: "
-                    + this.getPhase() + " vs " + containerPhase);
-            }
+        if (containerPhase < Integer.MAX_VALUE && this.getPhase() < Integer.MAX_VALUE && this.getPhase() != containerPhase) {
```

Comment:
should we break this into multiple lines?

Reply:
yes, I will refactor this line.

Reply:
I will delete these lines, for I think this judge is not necessary.